quantum/process_keycode/process_midi.h

```diff
@@ -35,7 +35,7 @@ typedef union {
     struct {
         uint8_t octave : 4;
         int8_t  transpose : 4;
-        uint8_t velocity : 4;
+        uint8_t velocity;
```

Comment:
Why the change here?

Reply:
The range of `velocity` is now 0-127 (as it should be), not 0-15.

Reply:
so, this could be `:7`, but simpler to just not.

Reply:
I originally figured readability was worth more than the single bit of memory that might be saved, but upon further consideration I think there is value to using `:7` here. Especially with how it enables more graceful handling of user-caused overflows. Now that users will be able to control `midi_config.velocity` directly there is nothing stopping them from going past 127 with something like an errant loop or misconfigured rotary encoder. It would probably be nicer of us to have the value overflow back to 0 once it tries to go past 127 than to allow out-of-range velocity values to be set in the first place.

I feel like it's best to protect users and other software and equipment further down the signal chain from invalid MIDI messages, which means either changing this line to `uint8_t velocity : 7;` or implementing some other check in `process_midi()` or `midi_send_noteon()`. Out of these options, `uint8_t velocity : 7;` is probably the simplest one